doc/rbd/rbd-mirroring.rst

```diff
@@ -0,0 +1,233 @@
+replication between clusters.  Mirroring is configured on a per-pool basis
+within peer clusters and can be configured to automatically mirror all pool
+images within a pool or only a specific subset of images.  Mirroring is
+configured using the ``rbd`` command.  The ``rbd-mirror`` daemon is responsible
+for pulling image updates from the remote, peer cluster and applying them to
+the image within the local cluster.
+
+.. important:: To use use RBD mirroring, you must have two Ceph clusters and
+   the ability for the ``rbd-mirror`` daemon to connect to both clusters
+   simultaneously.
+
+Pool Configuration
+==================
+
+The following procedures demonstrate how perform the basic administrative
+tasks to configure mirroring using the ``rbd`` command.  Mirroring is
+configured on a per-pool basis within the Ceph clusters.
+
+The pool configuration steps should be performed on both peer clusters. These
+procedures assume two clusters, named "local" and "remote", are reachable from
+a single host for clarity.
+
+See the `rbd`_ manpage for additional details of how to connect to different
+Ceph clusters.
+
+Enable Mirroring
+----------------
+
+To enable mirroring on a pool with ``rbd``, specify the ``mirror pool enable``
+command, the pool name, and the mirroring mode::
+
+        rbd mirror pool enable {pool-name} {mode}
+
+The mirroring mode can either be ``pool`` or ``image``:
+
+* **pool**:  When configured in ``pool`` mode, all images in the pool with the
+  journaling feature enabled are mirrored.
+* **image**: When configured in ``image`` mode, mirroring needs to be
+  `explicitly enabled on each image`_.
+
+For example::
+
+        rbd --cluster local mirror pool enable image-pool pool
+        rbd --cluster remote mirror pool enable image-pool pool
+
+Disable Mirroring
+-----------------
+
+To disable mirroring on a pool with ``rbd``, specify the ``mirror pool disable``
+command and the pool name::
+
+        rbd mirror pool disable {pool-name}
+
+When mirroring is disabled on the pool, any images where mirroring was
+explicitly enabled will also be disabled.
+
+For example::
+
+        rbd --cluster local mirror pool disable image-pool
+        rbd --cluster remote mirror pool disable image-pool
+
+Add Cluster Peer
+----------------
+
+In order for the ``rbd-mirror`` daemon to discover its peer cluster, the peer
+needs to be registered to the pool.  To add a mirroring peer Ceph cluster with
+``rbd``, specify the ``mirror pool peer add`` command, the pool name, and a
+cluster specification::
+
+        rbd mirror pool peer add {pool-name} {client-name}@{cluster-name}
+
+For example::
+
+        rbd --cluster local mirror pool peer add image-pool client.remote@remote
+        rbd --cluster remote mirror pool peer add image-pool client.local@local
+
+.. note:: The peer cluster name corresponds to a Ceph configuration file of the
+   same name (e.g. /etc/ceph/remote.conf)
```

Comment:
This seems to imply that `/etc/ceph/remote.conf` needs to be present on the host where we are running the `rbd` command. But ordinarily it would only be present on the remote cluster itself, right?